components/AccountDetails/TransactionHistory/TransactionHistory.tsx

```diff
@@ -55,6 +55,12 @@ function TransactionHistory() {
             return 'Bought Pre-API'
         }
 
+        const parts = transaction.reference.split('.')
+        let suffix = ''
+        if (parts.length === 2 && /^[0-9a-fA-F]{32}$/.test(parts[0]) && /^[0-9a-fA-F\-]{1,}$/.test(parts[1])) {
+            suffix = `(License for ${parts[0]})`
```

Comment:
```suggestion
            suffix = ` (License for ${parts[0]})`
```
Strictly speaking there should only be a space when there actually is a suffix